Blizzard.IO.RabbitMQ/NetqRabbitPublisher.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using Blizzard.IO.RabbitMQ.Entities;
+using Blizzard.IO.Core;
+using EasyNetQ;
+using EasyNetQ.Topology;
+
+namespace Blizzard.IO.RabbitMQ
+{
+    public class NetqRabbitPublisher<TData> : IRabbitPublisher<TData>, IPublisherWithMetadata<TData, RabbitMessageProperties>, IPublisher<TData>
+    {
+        private readonly IBus _netqBus;
+        private readonly ISerializer<TData> _serializer;
+        private readonly IExchange _destinationExchange;
+        private readonly IConverter<RabbitMessageProperties, MessageProperties> _rabbitMessagePropertiesToMessagePropertiesConverter;
+        private readonly string _routingKey;
+        private readonly bool _isAbstract;
+
+        public NetqRabbitPublisher(IBus netqBus, ISerializer<TData> serializer, RabbitExchange destinationExchange, bool isAbstract = false, string routingKey = "/")
+        {
+            _netqBus = netqBus;
+            _serializer = serializer;
+            _destinationExchange = _netqBus.Advanced.ExchangeDeclare(destinationExchange.Name.ToLower(),
```

Comment:
Why are you converting the exchange name to lower case leters?

Reply:
f